### DISCUSSION TITLE
请问 REALITY 中的 "spiderX": "" 留空或不写此参数时的情况

### DISCUSSION THREAD

**Discussion by chika0801** [XTLS/Xray-core#2385] [Q&A]
我阅读了 https://github.com/XTLS/REALITY 里面的说明

当` REALITY 客户端可以完美区分临时可信证书、真证书、无效证书，并决定下一步动作：`**收到真证书时，进入爬虫模式**

如果此时客户端配置中没写  `"spiderX": ""` 参数，或值是**留空**时。代码给的默认工作状态是 **爬虫模式工作**，且爬虫访问 Dest 网址的路径是 `"spiderX": "/"` 吗？ 

即 不写 `"spiderX"` 时= 写 `"spiderX": ""` = 写 `"spiderX": "/"` 请问正确吗？

**Comment by RPRX** [comment]
对

**Reply by chika0801** [reply]
感谢您解答
